### DISCUSSION TITLE
Why is the repo name mandatory if helm?

### DISCUSSION THREAD

**Discussion by mhagnumdw** [argoproj/argo-cd#10742] [Q&A]
I'm trying to understand some points of ArgoCD.

Why when creating a repo of type SSH, the name has this restriction: _name of the repository, mandatory for repositories of type helm_ ?

I would like to understand why. Thanks!

**Comment by reginaelyse** [comment]
I'm not exactly sure, maybe some else might have a better answer, but from looking at the code when `getHelmRepos` gets called (which also happens when helm repos are created) it requires a name.
